pkg/config/validation/validation.go

```diff
@@ -0,0 +1,49 @@
+package validation
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+)
+
+// FindEmptyStringsInConfigStruct finds empty strings in a struct.
+// in case the struct contains another struct, also this struct is checked.
```

Comment:
As the line above ends with a full stop I would expect this to be a new sentence, starting with a capital letter. But maybe the line above is supposed to end with a comma?

Reply:
adjusted.